website/docs/api/morphologizer.md

```diff
@@ -176,7 +176,7 @@ config.
 
 | Name           | Description                                                                                                                                                                                                                                                                                                                                                                                       |
 | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `get_examples` | Function that returns gold-standard annotations in the form of [`Example`](/api/example) objects. ~~Callable[[], Iterable[Example]]~~                                                                                                                                                                                                                                                             |
+| `get_examples` | Function that returns gold-standard annotations in the form of [`Example`](/api/example) objects. Must contain at least one annotated `Example`. ~~Callable[[], Iterable[Example]]~~                                                                                                                                                                                                              |
```

Comment:
The bit about labels vs. annotated examples is also true here, so I'd make a similar change to "must contain at least one example" everywhere.